lexicons/tools/ozone/communication/createTemplate.json

```
@@ -23,6 +23,10 @@
               "type": "string",
               "description": "Subject of the message, used in emails."
             },
+            "lang": {
+              "type": "string",
+              "description": "Message language."
+            },
```

There is a strong format for languages!
```suggestion
            "lang": {
              "type": "string",
              "format": "language",
              "description": "Message language."
            },
```

---

oh that's awesome! didn't know lexicons had that. is there a list of supported formats somewhere?